invenio_override/config.py

```diff
@@ -69,13 +95,13 @@
 THEME_HEADER_LOGIN_TEMPLATE = "invenio_override/accounts/header_login.html"
 """login page header"""
 
-THEME_FOOTER_TEMPLATE = "invenio_override/footer.html"
+THEME_FOOTER_TEMPLATE = "invenio_override/footer_mug.html"
```

Comment:
```suggestion
THEME_FOOTER_TEMPLATE = "invenio_override/footerhtml"
```
use the default for this repo, as we discussed

Reply:
yap, sorry i forgot to revert while testing it! Ill revert this.